synapse/tests/test_lib_lmdbslab.py

```diff
@@ -334,7 +334,7 @@ async def test_lmdbslab_max_replay(self):
 
             # Make sure that we don't confuse the periodic commit with the max replay log commit
 
-            with patch('synapse.lib.lmdbslab.Slab.COMMIT_PERIOD', 10):
+            with patch.dict(os.environ, {"SYN_SLAB_COMMIT_PERIOD": "10"}):
```

Comment:
``self.setTstEnvars`` should be used here to set a specific envar

Reply:
ok